identity/src/test/java/com/stripe/android/identity/viewmodel/IdentityScanViewModelTest.kt

```diff
@@ -49,6 +49,7 @@ internal class IdentityScanViewModelTest {
         mockFpsTracker,
         mockIdentityAnalyticsRequestFactory,
         mock(),
+        mock(),
```

Comment:
Would be nice to see tests for this, rather than just mocks.

Reply:
Happy to chat about this if you think it's worthwhile! The line you commented on is a `finishWithResult` function, were you suggesting that I write tests for `errorHandler`?